enterprise/payara.tooling/nbproject/org-netbeans-modules-payara-tooling.sig

```diff
@@ -1398,34 +1398,9 @@ meth public abstract org.netbeans.modules.payara.tooling.TaskEvent getEvent()
 meth public abstract org.netbeans.modules.payara.tooling.data.PayaraStatusCheck getType()
 meth public abstract org.netbeans.modules.payara.tooling.data.PayaraStatusCheckResult getStatus()
 
-CLSS public final !enum org.netbeans.modules.payara.tooling.data.PayaraVersion
+CLSS public final org.netbeans.modules.payara.tooling.data.PayaraVersion
```

Comment:
Another attempt to change an `enum` to something extensible. See [again](http://wiki.apidesign.org/wiki/Enum).